ui/pages/lock.tsx

```diff
@@ -255,7 +248,7 @@ export default function Lock() {
                       veNationBalance &&
                       veNationBalance.value.gt(ethers.utils.parseEther('1'))
                         ? 2
-                        : 8
+                        : 6
```

Comment:
Reduced # decimals as this caused overflow issues in the app. Also, in the component for locking tokens, this also causes the content to jump up and down depending on the number, which looks buggy

<img width="512" alt="image" src="https://user-images.githubusercontent.com/108498243/183981127-17626677-1a7a-4c9d-b464-98848b5c067d.png">

<img width="524" alt="image" src="https://user-images.githubusercontent.com/108498243/183981155-44101b98-75cc-4d41-a938-683c26ccfd8a.png">

Reply:
@southbears how come you are running it in dark mode?

Reply:
I'm using this Brave plugin because my eyes prefer dark mode ;)
https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh?hl=en